### PR TITLE
music: new D_DM2INT by BlueWorrior

### PR DESCRIPTION
**Pull request by Talon1024**

### CODE REVIEW

**Comment by mc776** [comment]
Does the track have a title?

**Comment by unithecutie** [comment]
It was already pushed; it's called "FTW You 100".

**Comment by Talon1024** [comment]
I don't know if that's the actual title, however. It was the name of the MIDI file BlueWorrior posted on Discord.

File: CREDITS-MUSIC [hunk 1]
```diff
@@ -53,7 +53,7 @@ Episode 4 - Double Impact
 --Phase 2--
 
 Title: "Fight for Freedom" by Korp, based on "Fountain Dance" by Tyler "Picklehammer" Pantella
-Intermission: "Burn The Bodies" by Tyler "Picklehammer" Pantella
```
Comment (mc776) [inline CREDITS-MUSIC:56]: Could you add this to the "Previously Featured" section?

**Comment by unithecutie** [comment]
Should we replace the "previously featured" section with stuff from 0.12.1 instead?

E1M3: "Mortal" by Jeremy "Vandalorian" Emerson
E1M5 + E2M6: "sit on a bench 'n release that tensh'n" by Jazz "jmickle" Mickle
E1M6: dave3d35.mid by jute
E1M7: Sewers by Discoholic and Samuel "Blastfrog" Oliver (moved to MAP12, not removed)
E1M9: "The Raging Sun" by Kevin "Velvetic" Martins and Jonathan "julnen" Martins (moved to E1M5, not removed)
E2M1: dave3d38.mid by jute
E2M2: "Animal Skin" by Kevin "Velvetic" Martins and Jonathan "julnen" Martins
E2M3: "Alone with Colors" by Kevin "Velvetic" Martins and Jonathan "julnen" Martins (moved to E1M3, not removed)
E2M4: "Let 'em Burn" by Kevin "Velvetic" Martins
E2M5: "Music for Freedoom 1" by Jeremy "Vandalorian" Emerson
E2M6: "sit on a bench 'n release that tensh'n" by Jazz "jmickle" Mickle
E2M7: "Music for Freedoom 3" by Jeremy "Vandalorian" Emerson
E2M8: ? by Anthony "Ajanddino" Pierce
E2M9: dave3d34.mid by jute
Phase 1 Title: "Fountain Dance" by Tyler "Picklehammer" Pantella
OPL Title: "I Hate Fredrik Johansson" by Toby "Tobester" Collins Jr.
Victory: "Ominous Theme" by Jared "BlackJar72" Blackburn
Bunny: "Look At The Bunnies" by RestlessRodent


Phase 2 + FreeDM Title: "Warpath" by Tyler "Picklehammer" Pantella
Phase 2 + FreeDM Intermission: "Burn the Bodies" by Tyler "Picklehammer" Pantella
MAP04: "Soviet Porno" by Jeremy "Vandalorian" Emerson (moved to MAP10, not removed)
MAP06: "Dark Trail" by Tyler "Picklehammer" Pantella
MAP07: "Riders of Hell" by Tyler "Picklehammer" Pantella (moved to DM05, not removed)
MAP10: "Death's Embrace" by Tyler "Picklehammer" Pantella
MAP11: dave3d02.mid by jute
MAP12: untitled by Bradley "BobFromReboot" Lavigne
MAP21: "Violence Culture" by Jeremy "Vandalorian" Emerson
MAP22: "Mists Over the Mire" by Tyler "Picklehammer" Pantella
MAP23: dave3d09.mid by jute
MAP24: "Balancing Act" by Andrew Bassett
MAP26: "Crawl" by Jeremy "Vandalorian" Emerson
MAP30: "Paranoia" by Tyler "Picklehammer" Pantella

DM03: "Music for Freedoom 9" by Jeremy "Vandalorian" Emerson
DM05: map01.mid by Ralph "Ralphis" Vickers (moved to MAP07, not removed)
DM06: "The Heroic Battle" by Tyler "Picklehammer" Pantella
DM07: dave3d30.mid by jute
DM08: Dave3d13.mid by jute
DM12: dave3d24.mid by jute
DM13: e1m1.mid by Ralph "Ralphis" Vickers
DM18: dave3d06.mid by jute
DM19: dave3d27.mid by jute
DM20: dave3d16.mid by jute
DM21: dave3d12.mid by jute
DM23: dave3d27.mid by jute
DM25: dave3d20.mid by jute
DM30: dave3d01.mid by jute
DM32: "Munt Battle" by Mark "TheMisterCat" McGill-Smith

**Review by mc776 — APPROVED** [review]

**Comment by mc776** [comment]
Approved but given the *drastic* change in vibe I'll let this sit for a bit longer in case anyone's got a second opinion.

**Comment by mc776** [comment]
> Should we replace the "previously featured" section with stuff from 0.12.1 instead? If I've missed anything, please let me know.

Would it be better to have this at the end of the existing music credits file or a new file?

**Comment by unithecutie** [comment]
I think leaving it as is with it at the bottom of the existing music credits file would be a better idea; I have an entire 0.12.1 document to reference if needed.

**Pull request merged**